resources/quality/KOONOVO_ELF_draft.inst.cfg

```diff
@@ -0,0 +1,61 @@
+[general]
+version = 4
+name = Draft
+definition = KOONOVO_ELF
+
+[metadata]
+setting_version = 16
+type = quality
+quality_type = draft
+weight = -2
+global_quality = True
+
+[values]
+acceleration_enabled = True

```

Comment:
These values are the same everywhere. It's better to just leave them in your printer definition.

Reply:
> These values are the same everywhere. It's better to just leave them in your printer definition.
definition in KOONOVOELF.def.json